Directory.Packages.props

```diff
@@ -1,13 +1,11 @@
 <Project>
-
   <!-- Azure -->
   <ItemGroup>
-    <PackageVersion Include="Microsoft.Azure.DurableTask.Core" Version="2.13.0" />
+    <PackageVersion Include="Microsoft.Azure.DurableTask.Core" Version="2.16.1" />
```

Comment:
Does this need to be upgraded? I prefer to leave dependencies of a library at a minimum for what the code base needs directly. End users can then decide to pin transitive dependencies at a higher version if they need.

This lets this library capture the widest customer base possible: if a project for some reason cannot update a transitive dependency, this will not force them to.